cardano-cli/src/Cardano/CLI/Shelley/Run/Genesis.hs

```diff
@@ -1011,19 +989,19 @@ updateTemplate (SystemStart start)
         shelleyGenesis = template
           { sgSystemStart = start
           , sgMaxLovelaceSupply = fromIntegral $ nonDelegCoin + delegCoin
-          , sgGenDelegs = shelleyDelKeys
-          , sgInitialFunds = Map.fromList
+          , sgGenDelegs = shelleyDelKeys -- TODO should this be a ListMap?
```

Comment:
This should not be a `ListMap`. See: https://github.com/input-output-hk/cardano-ledger/pull/2891#issuecomment-1172541619
```suggestion
          , sgGenDelegs = shelleyDelKeys
```

Reply:
Removed comment